src/transformers/__init__.py

```diff
@@ -934,6 +936,7 @@
             "MODEL_FOR_SPEECH_SEQ_2_SEQ_MAPPING",
             "MODEL_FOR_TABLE_QUESTION_ANSWERING_MAPPING",
             "MODEL_FOR_TOKEN_CLASSIFICATION_MAPPING",
+            "MODEL_FOR_UNIVERSAL_SEGMENTATION_MAPPING",
```

Comment:
I'm not sure there's a need for this mapping, as OneFormer is very specific and will probably be the only class included in this mapping.